stripe/build.gradle

```diff
@@ -13,7 +13,6 @@ dependencies {
     compile 'com.android.support:support-v4:' + android_support_version
     compile 'com.android.support:appcompat-v7:' + android_support_version
 
-    compile()
```

Comment:
It was an accidentally-included empty line, sort of like leaving an if() { } line in your java code. Build files don't get changed that often, and in a particularly tricky PR, it's possible that they can be overlooked, which must have happened here.

It doesn't technically hurt anything, but it's useless, so I'm taking it out now that I noticed it.